operator/cmd/mesh/install.go

```diff
@@ -187,3 +199,46 @@ func InstallManifests(setOverlay []string, inFilenames []string, force bool, dry
 
 	return saveIOPToCluster(reconciler, iopStr)
 }
+
+func handleProfileFromCluster(iArgs *installArgs) error {
+	// Read the IOP from Kubernetes
+	kubeConfigFlags := &genericclioptions.ConfigFlags{
+		Context:    strPtr(""),
```

Comment:
why are these all empty, and do we need to initialize a new kubeconfig instead of using the one we already create for install?